public/app/plugins/panel/barchart/utils.test.ts

```diff
@@ -166,13 +168,16 @@ describe('BarChart utils', () => {
     });
 
     it('will warn when there is no string or time field', () => {
-      const df = new MutableDataFrame({
```

Comment:
needed to change this because MutableDataFrame will change to numbers if not specified